src/test_l1_dsHdmiIn.c

```diff
@@ -400,19 +412,25 @@ void test_l1_dsHdmiIn_negative_dsHdmiInGetStatus(void) {
     dsHdmiInStatus_t status;
 
     // Step 1: Call dsHdmiInGetStatus() without initializing the HDMI input sub-system
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     UT_ASSERT_EQUAL(dsHdmiInGetStatus(&status), dsERR_NOT_INITIALIZED);
+    #endif
 
     // Step 2: Initialize the HDMI input sub-system using dsHdmiInInit()
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     // Step 3: Call dsHdmiInGetStatus() with NULL as the parameter
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     DS_ASSERT_AUTO_TERM_NUMERICAL(dsHdmiInGetStatus(NULL), dsERR_INVALID_PARAM);
```

Comment:
In the else case, we need to handle the expected dsERR_GENERAL

Reply:
done

Reply:
done